coffee_lib/src/types/mod.rs

```diff
@@ -86,6 +86,7 @@ pub mod request {
 // Definition of the response types.
 pub mod response {
     use serde::{Deserialize, Serialize};
+    use std::fmt;
```

Comment:
we can fix the import here?
```suggestion
    use std::fmt;
    
    use serde::{Deserialize, Serialize};
```